doc/cdi-operator-olm.md

```diff
@@ -0,0 +1,26 @@
+# OLM intergartion
+##OLM Overview
+https://github.com/kubevirt/kubevirt/blob/master/docs/devel/olm-integration.md
+
+
+## CDI OLM manifests 
+1. in order to obtain olm bundle manifest run the following command
+```bash
+DOCKER_REPO=<repo> DOCKER_TAG=<docker tag> PULL_POLICY=<pull policy> VERBOSITY=<verbosity> CSV_VERSION=<CSV version> QUAY_NAMESPACE=<namespace> QUAY_REPOSITORY=<application name> make manifests
+```
+The generated final olm manifests will be located in _out/manifests/release/olm/bundle/_ directory
+
+Note: there is a structure of operator related manifest
+- manifests/release - contains operator manifests that can be deployed without olm 
+- manifests/olm - contains additional auxilary manifests that are required when deploying with olm and with olm marketplace
+- manifests/olm/bundle - contains olm bundle  that is to be pushed to quay.io and consumed by marketplace operator
+2. in order to verify generated manifests run the follwoing command
+```bash
+make olm-verify
+```
+3. in order to push the generated verified manifests to quay.io run 
+```bash
+CSV_VERSION=<CSV version>  QUAY_USERNAME=<quay account username> QUAY_PASSWORD=<quay account password> QUAY_NAMESPACE=<namespace> QUAY_REPOSITORY=<application name> make olm-push
```

Comment:
We need travis integration for this?

Reply:
So this will publish to quay in addition to dockerhub then?

Reply:
yes

Reply:
@awels  and @mhenriks  more update to travis manifest is required in order to olm-verify and olm-push olm bundle manifests - working on it

Reply:
we will integrate olm-push into travis once csv update support is ready